### PR TITLE
feat: images and input DONE

### PR DESCRIPTION
**Pull request by maitrakhatri**
Please review the images and input component

https://user-images.githubusercontent.com/54719132/151706225-a8b126c4-fed8-4ae3-b9ef-8252ca80b144.mp4

Screenshot 2022-01-30 at 21-02-08 input

### CODE REVIEW

File: image/image.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@import url(../style.css);
+
+.img-res {
+    width: 100%;
+    height: auto;
+}
+
+.img-round-250px {
+    border-radius: 50%;
```
Comment (rohanmathur91) [inline image/image.css:9]:
```suggestion
make utility class for border-radius and object-fit.
````

**Review by rohanmathur91 — APPROVED** [review]

File: image/image.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    width: 250px;
+    height: 250px;
+    object-fit: cover;
```
Comment (rohanmathur91) [inline image/image.css:12]:
```suggestion
    make utility class for border-radius and object-fit.
````

**Review by rohanmathur91 — APPROVED** [review]

**Review by anupkgurung — APPROVED** [review]
looks good.

File: image/image.css (hunk 2 above)
Comment (maitrakhatri) [reply image/image.css:12]: > 

yes !! I'll make a utility class for common styles at the end and make a few changes

**Pull request merged**